ucp/_libs/ucx_api.pyx

```diff
@@ -326,6 +332,169 @@ cdef class UCXContext(UCXObject):
         ucp_context_print_info(self._handle, text_fd)
         return decode_text_fd(text_fd)
 
+    def map(self, mem):
+        return UCXMemoryHandle.map(self, mem)
+
+    def alloc(self, size):
+        return UCXMemoryHandle.alloc(self, size)
+
+
+cdef class PackedRemoteKey:
```

Comment:
I would be good with a description of what this class represent and its parameters.